cmd/kube-scheduler/app/server.go

```diff
@@ -127,6 +127,7 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.config.SchedulerName, "scheduler-name", o.config.SchedulerName, "Name of the scheduler, used to select which pods will be processed by this scheduler, based on pod's \"spec.SchedulerName\".")
 	fs.StringVar(&o.config.LeaderElection.LockObjectNamespace, "lock-object-namespace", o.config.LeaderElection.LockObjectNamespace, "Define the namespace of the lock object.")
 	fs.StringVar(&o.config.LeaderElection.LockObjectName, "lock-object-name", o.config.LeaderElection.LockObjectName, "Define the name of the lock object.")
+	fs.Int32Var(&o.config.CacheDebugResyncInterval, "cache-debug-resync-interval", o.config.CacheDebugResyncInterval, "The interval in seconds local cache is rebuilt and compared against existing cache (Only used for debug).")
```

Comment:
nit: try `The interval in seconds at which the local cache is rebuilt and compared against the existing cache. This should only be used for debugging.`

You may also wish to clarify *why* it is only for debugging.

Reply:
If you use metav1.Duration, you can delete "in seconds"